src/Sarif/GrammarFiles/Sarif.g4

```diff
@@ -104,7 +104,7 @@ toolInfo :
         three values for product version in the VS_VERSION_INFO structure. If the tool is a .NET
         application, this value SHOULD be the first three dotted version values of AssemblyVersion.
         }
-		@pattern {^\d+\.\d+\.\d+\.(-([A-Za-z0-9\-\.]+))?(\+[A-Za-z0-9\-\.]+)?$}
+		@pattern {[0-9]+\\.[0-9]+\\.[0-9]+(-[0-9A-Za-z-]+)?(\\+[0-9A-Za-z-]+)?}
```

Comment:
I believe our spec says that version SHOUDL be a semantic ver but does not need to be. Should we remove validation entirely? I guess this is a place where a warning in a SARIF validation service would be helpful. In fact, this is a good definition of what belongs there: analysis where the SHOULD standards are getting compromised, surfaced as warnings.